docs/source/getting-started/networking-basics.md

```diff
@@ -41,6 +42,10 @@ port.
 
 ## Set the Proxy's REST API communication URL (optional)
 
+Note to open issue about the following paragraph. The wording is a little unclear. 
+Is there a second port that the Hub service uses? Or is the api_url the only
+available port here?
```

Comment:
The Hub exclusively talks to the proxy via its REST API URL. This is the only URL by which the proxy can be configured.